### PR TITLE
Refactor event severity code base to follow new pattern 

### PR DESCRIPTION
**Pull request by Mloweedgar**

### CODE REVIEW

File: src/Events/EventSeverity/index.js [hunk 1]
```diff
@@ -102,25 +205,80 @@ class EventSeverities extends Component {
...
+      okText: 'Yes',
+      okType: 'danger',
+      cancelText: 'No',
+      onOk() {
+        deleteEventSeverity(
+          item._id, // eslint-disable-line
+          () => notifySuccess('Focal Person was archived successfully'),
+          () =>
+            notifyError(
+              'An error occurred while archiving Focal Person, Please contact your system Administrator'
```
Comment (BenMaruchu) [inline src/Events/EventSeverity/index.js]: Supposed to be Severity not focal person

File: src/Events/EventSeverity/index.js [hunk 2]
```diff
@@ -102,25 +205,80 @@ class EventSeverities extends Component {
...
+  showArchiveConfirm = item => {
+    confirm({
+      title: `Are you sure you want to archive this record ?`,
+      okText: 'Yes',
+      okType: 'danger',
+      cancelText: 'No',
+      onOk() {
+        deleteEventSeverity(
+          item._id, // eslint-disable-line
+          () => notifySuccess('Focal Person was archived successfully'),
```
Comment (BenMaruchu) [inline src/Events/EventSeverity/index.js]: S
hould be updated to severity

File: src/Events/EventSeverity/index.js (hunk 1 above)
Comment (BenMaruchu) [inline src/Events/EventSeverity/index.js]: Here also

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
Hello @Mloweedgar  can you work on the above reviews

File: src/Events/EventSeverity/index.js [hunk 3]
```diff
@@ -77,9 +136,9 @@ class EventSeverities extends Component {
   /**
    * @function
    * @name handleEdit
-   * @description Handle on Edit action for list item
+   * @description Handle on Edit severity for list item
    *
-   * @param {object} eventSeverity event severity to be edited
+   * @param {object} eventSeverity eventSeverity to be edited
```
Comment (navish) [inline src/Events/EventSeverity/index.js]: Aside from variable names all other words are normal, do not convert them to camel case 
e.g. `@param {object} eventSeverity Event severity to be edited`

**Review by navish — CHANGES_REQUESTED** [review]
Aside from variable names all other words are normal, do not convert them to camel case

**Review by BenMaruchu — APPROVED** [review]

**Pull request merged**